sbt-app/src/sbt-test/plugins/plugin-dependency-override-test/test

```diff
@@ -0,0 +1 @@
+> check
```

Comment:
We run scripted tests are batched, and each test is meant to be self-contained.
I think the prep test can be merged into here by `copy-file` (https://www.scala-sbt.org/1.x/docs/Testing-sbt-plugins.html) that creates `project/plugins`.

Reply:
could you expand on that?
I get that you want it to all be in one test directory, I'm just unsure what your suggested solution is.
Are you saying: 

1. all the subprojects from the setup build should be included in the test build base directory
2. they should be included in the project directory, or
3. they should be included in the plugins.sbt

If its any of those I don't see how it would come together. 
Are you saying that I should overwrite the plugins.sbt once the plugins have been published so that the test can use it to import them? Is that even possible?

Reply:
> Are you saying that I should overwrite the plugins.sbt once the plugins have been published so that the test can use it to import them? Is that even possible?

Yea, so I guess what I was thinking is something like:

```scala
> plugin/publishLocal
> plugin1/publishLocal
> plugin2/publishLocal
> dependentplugin/publishLocal

$ copy-file changes/build.sbt build.sbt
$ copy-file changes/plugins.sbt project/plugins.sbt

> reload
> check
```

Reply:
At long last this is done